### PR TITLE
Update Aleo instructions and Leo for commits and hashes.

### PR DESCRIPTION
**Pull request by acoglio**
Matches https://github.com/AleoHQ/snarkVM/pull/1576.

### CODE REVIEW

File: documentation/aleo/04_opcodes.md [hunk 1]
```diff
@@ -411,21 +429,215 @@ The instruction will halt if the given `String` or `Struct` value exceeds the 64
...
+
+***
+
+### `commit.ped128`
+
+[Back to Top](#table-of-standard-opcodes)
+
+#### Description
+
+Computes a Pedersen commitment up to a 128-bit input in `first`, and some randomness in `second`, storing the commitment in `destination`. Randomness should always be a `Scalar` value, and the produced commitment will always be a `Group` value.
```
Comment (bendyarm) [inline documentation/aleo/04_opcodes.md:452]: "will always be a Group" --> "is a Field"

Comment (bendyarm) [reply documentation/aleo/04_opcodes.md:452]: For both `commit.ped64` and `commit.ped128`, the produced commitment is a `Field` value.

**Review by bendyarm — CHANGES_REQUESTED** [review]

File: documentation/aleo/04_opcodes.md (hunk 1 above)
Comment (acoglio) [reply documentation/aleo/04_opcodes.md]: Done.

**Comment by acoglio** [comment]
I just pushed a commit that updates things on the Leo side, besides Aleo instructions.

**Review by bendyarm — APPROVED** [review]
Looks good.

**Pull request merged**